packages/common/src/project/types.ts

```diff
@@ -2,12 +2,13 @@
 // SPDX-License-Identifier: Apache-2.0
 
 import {SubqlDatasource} from '@subql/types';
+import {SubqlTerraDatasource} from '@subql/types-terra';
```

Comment:
`@subql/common` is used by `@subql/node`, therefore by introducing `@subql/types-terra` it contaminate dependencies of  `@subql/node`